bd_spot_wrapper/spot_wrapper/spot.py

```diff
@@ -836,6 +919,19 @@ def get_camera_intrinsics(self, source, quality=None, pixel_format=None):
         cam_intrinsics = image_response.source.pinhole.intrinsics
         return cam_intrinsics
 
+    def get_camera_intrinsics_as_3x3(self, camera_intrinsics) -> np.ndarray:
```

Comment:
I think this would be something like

```
fx=intrinsics.focal_length.x,
fy=intrinsics.focal_length.y,
px=intrinsics.principal_point.x,
py=.intrinsics.principal_point.y
K = np.array([[fy, 0, px], [0, fy, py], [0, 0, 1]])
```

Reply:
awesome. Thanks for sharing this.